composer.json

```diff
@@ -2,7 +2,18 @@
     "name": "cakephp/cakephp",
     "description": "The CakePHP framework",
     "type": "library",
-    "keywords": ["framework"],
+    "keywords": [
+        "framework",
+        "mvc",
+        "rapid-development",
+        "conventions over configuration",
+        "dry",
+        "orm",
+        "form",
+        "validation",
+        "rest-api",
```

Comment:
The framework is the sum of those sub packages (well more than just that) and thus both can contain the keywords that fit in what they provide.